### ISSUE TITLE
[Bug]: Navigating to about:blank fails non-headless (regression in 14.0.0)

### ISSUE THREAD

**Issue by pasieronen** [puppeteer/puppeteer#8334]
### Bug description

Navigating to `about:blank` when not headless works fine with puppeteer@13.7.0, but fails with 30s timeout on 14.0.0:

```
const puppeteer = require('puppeteer');
(async () => {
  const browser = await puppeteer.launch({headless: false});
  const page = await browser.newPage();
  await page.goto('about:blank');
  console.log('ok');
  await browser.close();
})();
```

Headless mode works also on 14.0.0.

### Puppeteer version

14.0.0

### Node.js version

16.15.0

### npm version

8.5.5

### What operating system are you seeing the problem on?

macOS

**Comment by OrKoN** [comment]
Hey, thanks for the report! Unfortunately, it's a bug upstream (Chromium M102) and we added a workaround for it yesterday https://github.com/puppeteer/puppeteer/pull/8330 (to be released soon). The upstream issue is also fixed and it will be back-merged into the m102 branch (crbug.com/1324138) and then hopefully we are able to re-roll the Chromium binary and remove the workaroound.

**Comment by OrKoN** [comment]
The workaround is released in 14.1.0.

**Comment by pasieronen** [comment]
Thanks, this seems to work in 14.1.0 and 15.3.1 (latest), closing issue.
